saltstack/base/salt/luks/bin/create-luks-volume.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -ex -o pipefail
+
+: ${SECRET_ENCRYPTION_KEY_SOURCE:? required}
+
+export LUKS_BACKING_FILE_DEFAULT_SIZE="100MiB"
+
+if mountpoint /mnt/luks; then
+  echo "Volume already mounted at mount point... Exiting LUKS volume creation!"
+  exit
+fi
+
+if [[ -e /etc/luks/luks && $(stat -c "%a" /etc/luks/luks | tr -d '\n') == 600 ]]; then
+  # Set the backing file to the specified size if it exists
+  truncate -s "$LUKS_BACKING_FILE_DEFAULT_SIZE" /etc/luks/luks
```

Comment:
Oh, I guess this is because `saltstack/base/salt/luks/init.sls` was supposed to pre-create the LUKS image file placeholder? I still think it is better to create the file here.